Makefile.PL

```diff
@@ -58,7 +58,7 @@ sub configure
 
   if (eval { ExtUtils::MakeMaker->VERSION(6) }) {
     push @moreopts, AUTHOR => 'Marcus Holland-Moritz <mhx@cpan.org>';
-    push @moreopts, ABSTRACT_FROM => 'PPPort_pm.PL';
+    push @moreopts, ABSTRACT_FROM => 'PPPort.pm';
```

Comment:
this is incorrect as this file is not part of the list

Reply:
Why does this line need to be changed at all?
But if the abstract can't be read from this file, it can simply be hardcoded as `ABSTRACT => 'Perl/Pollution/Portability'`.

Reply:
That's the idea use a static file somewhere, that could be updated on `make dist` 
not sure what is this file `Perl/Pollution/Portability`? where does it live? does it exist?
it's mention in `RealPPPort_xs.PL` but not sure what generates it?